renovate/defaults.json5

```diff
@@ -112,6 +112,19 @@ and/or use the pre-commit hook: https://github.com/renovatebot/pre-commit-hooks
     }
   ],
 
+  /*************************************************
+  ***** Language-specific configuration options ****
+  **************************************************/
+
+  // *****  ATTENTION  WARNING  CAUTION  DANGER  ***** //
+  // Go versions 1.21 and later will AUTO-UPDATE based on module
+  // requirements.  ref: https://go.dev/doc/toolchain  Fedora 40+
+  // disable this feature but that doesn't affect other distros,
+  // nor the behavior of the public renovate service.  The only
+  // way to fully disable these auto-updates in renovate is through
+  // the forced static-version below:
+  "constraints": {"go": "1.20"},
```

Comment:
lets hope this helps temporarily, but we will need to upgrade sooner or later anyway

Reply:
Yes, and I'm intending to add a warning comment to p|b|s `go.mod` as well.  But wanted to get this wording approved and in-place first.